### ISSUE TITLE
OBS wiki content moved link url broken

### ISSUE THREAD

**Issue by sjb8100** [AppImage/AppImageKit#932]
This wiki page tells us to go look at:
https://github.com/AppImage/AppImageKit/wiki/Using-Open-Build-Service#working-examples

> Content moved to https://docs.appimage.org/blob/master/source/packaging-guide/other-resources/obs.rst 

Follow that link get 404: Not Found, glad we kept the old content at the wiki :-)

**Comment by TheAssassin** [comment]
Is it too much asked to [search for `OBS`](https://docs.appimage.org/search.html?q=OBS&check_keywords=yes&area=default) using the provided search bar?

https://docs.appimage.org/packaging-guide/hosted-services/opensuse-build-service.html

**Comment by sjb8100** [comment]
Is that the answer we want here? This is the first taste for me of the AppImage community.

https://docs.appimage.org/blob/master/source/packaging-guide/other-resources/*
does not equal
https://docs.appimage.org/packaging-guide/hosted-services/*

Shouldn't we fix the wiki link?

**Comment by TheAssassin** [comment]
You didn't mention the location, and I couldn't find it from a quick search (tbh, I didn't spend more than a minute on that). Sorry for being a bit rude. The wiki is editable, can you please fix the link?

I'm a bit curious who's put `docs.appimage.org/blob/...` in the wiki. That link has never worked.

**Comment by sjb8100** [comment]
I admit I posted it and directly reviewed it, within 15 seconds I added the location to issue. Perhaps GitHub took a minute to update.

Either way thank you for the apology. While I appreciate quick response to an issue. I think there is a balance between quality of the response and quickness. I will edit the wiki with the link you provided.

**Comment by TheAssassin** [comment]
I've been a bit annoyed ever since I set up the docs due to this "wiki and docs in parallel is annoying to maintain" issue you start to get, hence I get a bit snippy with such issues. I'll try to be a bit more relaxed.

**Comment by sjb8100** [comment]
Keep in mind lol, I don't know that. Hey I am on your side, I was trying to find your docs anyway. Cheers!

**Comment by TheAssassin** [comment]
Yes, I know, now at least. I hope the docs will be helpful. If you ever find a bug or want to contribute, be my guest. We can provide more insights if needed, too, e.g., on IRC (#appimage on Freenode).

**Comment by sjb8100** [comment]
OK, I updated the link. So we can consider that done.

Since I might still have your eyes. I was coming here to investigate if anyone has even tried making a Unity 3D app image. Unity 3d builds to linux. I did search github issues and now the new docs for unity. Only getting the ubuntu unity which is different. I can build and use unity editor on linux. That is not the question it is how to take the game I built in Unity and then turn that into an AppImage. I know try it and see, but I was looking pointers first. Any tips where to post this question if I still have your eyeballs?

Ref: https://docs.appimage.org/search.html?q=unity&check_keywords=yes&area=default
Ref: https://docs.unity3d.com/Manual/BuildSettings.html (linux)

**Comment by TheAssassin** [comment]
I've never heard of anyone providing Unity 3D AppImages. With commercial software like that, it's always a bit tricky license wise -- distributing as AppImage may be violating different laws, mostly copyright related.

Just make a normal build of your Unity based application, and start with linuxdeploy. There's an extensive guide available [here](https://docs.appimage.org/packaging-guide/from-source/native-binaries.html). linuxdeploy will collect all your app's dependencies (shared libraries), and can produce an AppImage. Depending on how well your build system is at installing Unity resources and Unity is at loading resources relative to the main binary (i.e., relocatable), you might get away by just following the process documented. If not, there might be some potential for making a Unity plugin for linuxdeploy.

**Comment by probonopd** [comment]
The original poster states in https://discourse.appimage.org/t/improving-wiki-and-github-dev-ux/1105 that his concern was not properly addressed in this issue, but the issue was closed nevertheless. Hence, reopening.

**Comment by probonopd** [comment]
https://github.com/AppImage/AppImageKit/wiki/Using-Open-Build-Service#working-examples says in one section:

> Content moved to https://docs.appimage.org/packaging-guide/hosted-services/opensuse-build-service.html

https://docs.appimage.org/packaging-guide/hosted-services/opensuse-build-service.html does not give a 404 for me. It shows "Using the Open Build Service".

So, technically the issue described in the first post seems to have been resolved.

**Comment by probonopd** [comment]
> Since I might still have your eyes. I was coming here to investigate if anyone has even tried making a Unity 3D app image. 

I don't think anyone from the AppImage project has tried doing this. I know that an installer for Unity 3D is being distributed as an AppImage but that's about all I know about it. I don't even know what it is supposed to do.

> Any tips where to post this question if I still have your eyeballs?

In the Unity 3D forums I guess. Sorry we can't help. We only provide the tools for application authors to make AppImages, we don't know anything about most of the applications that are being distributed in AppImage format.

Does this appropriately answer your questions @sjb8100 so that we may close this issue?
